packages/react/src/components/Footer/__stories__/Footer.stories.js

```diff
@@ -2,4 +2,9 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import Footer from '../Footer';
 
-storiesOf('Footer', module).add('Default', () => <Footer />);
+// import './footer.scss';
+import '../../../../../styles/scss/components/footer/_index.scss';
```

Comment:
@photodow You should be able to avoid having to do this by importing your SCSS into a component
```suggestion
import '@ibmdotcom/styles/scss/components/masthead/_masthead.scss';
```

Reply:
@kennylam To the component, or the story where it is currently? This requires link `@ibmdotcom/styles`, correct?

Reply:
@photodow yeah if you link the packages you can import the css into the component with just a `@ibmdotcom/../../`. Then you're set for use in storybook as well.

Reply:
So I think I updated it correctly. Let me know if I misunderstood. On another note, this requires extra documentation. As dirty as the original was (I think we could clean it up a little more) it allows contributors to get going without having to link everything up, but this is good for now. It's not like we have a hundred contributors or anything right now. :P